Model/CategoryManagerInterface.php

```diff
@@ -27,11 +27,11 @@
     public function getRootCategory($context);
 
     /**
-     * @param bool|true $loadChildren
+     * @param bool $loadChildren
      *
      * @return CategoryInterface[]
      */
-    public function getRootCategories($loadChildren);
+    public function getRootCategories($loadChildren = true);
```

Comment:
No break, because this methods was introduced one commit ago ;) 

Btw. we are talking about the master branch.